pynestml/utils/model_installer.py

```diff
@@ -61,23 +73,31 @@ def install_nest(target_path: str, nest_path: str) -> None:
     else:
         shell = False
 
+    # remove CMakeCache.txt if exists
+    cmakeCache = os.path.join(target_path, "CMakeCache.txt")
+    if os.path.exists(cmakeCache):
+        os.remove(cmakeCache)
+
     # first call cmake with all the arguments
     try:
-        result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT, shell=shell,
-                                       cwd=str(os.path.join(target_path)))
+        result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT,
+                                       shell=shell, cwd=str(os.path.join(target_path)))
     except subprocess.CalledProcessError as e:
-        raise GeneratedCodeBuildException('Error occurred during \'cmake\'! More detailed error messages can be found in stdout.')
+        raise GeneratedCodeBuildException(
+            'Error occurred during \'cmake\'! More detailed error messages can be found in stdout.')
 
     # now execute make all
     try:
         subprocess.check_call(make_all_cmd, stderr=subprocess.STDOUT, shell=shell,
                               cwd=str(os.path.join(target_path)))
     except subprocess.CalledProcessError as e:
-        raise GeneratedCodeBuildException('Error occurred during \'make all\'! More detailed error messages can be found in stdout.')
+        raise GeneratedCodeBuildException(
+            'Error occurred during \'make all\'! More detailed error messages can be found in stdout.')
```

Comment:
```suggestion
        msg = "Error during 'make all'. More detailed error messages can be found in stdout."
        raise GeneratedCodeBuildException(msg)
```

Reply:
Again, those lines exceed the rule of 120 max-length.

Reply:
My line 94 has 90 characters, my line 95 has 47.